feature-libs/organization/administration/components/shared/organization-form/organization-form.component.html

```diff
@@ -3,15 +3,15 @@
     <button actions class="button primary">
       {{ 'organization.save' | cxTranslate }}
     </button>
-    <button actions class="link" routerLink="../" type="button">
+    <a actions class="link" routerLink="../" type="button">
```

Comment:
that causes strange redirection (look e.g. unit edit -> cancel)

Reply:
fixed